Add incident commander role + more steps to support process

choldgraf opened the pull request:
### Context

Our current support process is under-specified given the different kinds of support issues that we may get. This PR adds a few major concepts to our Support process documentation:

- **Change Requests**, **Incidents**, and **Guidance Requests** are a way of categorizing and prioritizing support tickets
- **Incident Response process** is a process we'll follow to resolve incidents
- **Non-incident response process** is a process for triaging and prioritizing Change Requests

It also adds some extra contextual information, terminology, etc to help us get on the same page.

### What are we missing

- [ ] There is not a clear picture of how support-related work items get routed to specific experts on our team
- [ ] ...anything else we're missing?

ref: https://github.com/2i2c-org/infrastructure/issues/1068#issuecomment-1123996834

also related to: https://github.com/2i2c-org/docs/pull/143 and https://github.com/2i2c-org/infrastructure/issues/1118

closes https://github.com/2i2c-org/infrastructure/issues/1154 closes https://github.com/2i2c-org/infrastructure/issues/1155

yuvipanda commented on projects/managed-hubs/support.md:
```diff
@@ -1,118 +1,217 @@
+  - They assess whether they can resolve it quickly, and potentially do so.
+  - If they cannot resolve it, then we raise this support issue with our engineering team.
+- If the issue is an {term}`Incident`
+  - We will prioritize resolving it over everything else.
```
I like this sentence!

yuvipanda commented on projects/managed-hubs/support.md:
```diff
@@ -1,118 +1,217 @@
-:::
+  :::{note}
+  We currently keep this term intentionally vague, and ask that communities are respectful of our time when making change requests.
+  We are investigating the support budget that we should give to each community, and will update here when we have specific numbers in mind. 
```
<3

yuvipanda commented on projects/managed-hubs/support.md:
```diff
@@ -1,118 +1,217 @@
-  with a short summary of the situation and/or add links to relevant discussions
-  from other platforms.
+Community Representative
+  A person that has the authority to speak on behalf of the communities we work with, and is a point-of-contact for support.
```
We should indicate that you can have multiple community representatives

choldgraf replied:
```suggestion
  A person that has the authority to speak on behalf of the communities we work with, and is a point-of-contact for support.
  Communities may have multiple community representatives.
```

yuvipanda commented on projects/managed-hubs/support.md:
```diff
@@ -1,118 +1,217 @@
+  See [](roles:community-representative) for more information.
 
-## How to prioritize support requests
+Post Mortem
```
IMO 'incident report' is a better term and we should use that consistently rather than post mortem

choldgraf replied:
whoops - yep agree

yuvipanda commented on projects/managed-hubs/support.md:
```diff
@@ -1,118 +1,217 @@
+3. **If you can resolve in 30 minutes**, try doing so.
+4. **If you cannot resolve in 30 minutes**, ping our engineering team and our Project Manager in the {guilabel}`#support-freshdesk` channel so that they are aware of the incident.
+5. **Investigate and resolve the incident**. Follow the structure of the incident issue you opened in the step above.
+6. **Communicate every few hours**. Provide periodic updates to communities as we attempt to resolve the incident. Here is a template to get started:
```
Who does this? It could be the support steward but I think it should also be ok to pass it on to an 'incident commander' for a particular incident.

choldgraf replied:
I think for this we'd need to define a new role, and incorporate that role into this process more generally. Do you think now is the right time to do that? (I am happy to put in the cycles to do this if you think it's a good time)

yuvipanda replied:
ya I think now is the right time to do that. Wanna get on a call to hash it out? Doesn't need to be part of this PR tho

choldgraf commented on projects/managed-hubs/support.md:
````diff
@@ -1,118 +1,217 @@
+
+1. **Acknowledge the incident**. Communicate with the Community Representative that there is an incident. Here is a template to get started:
+   
+   ```
````
If that is possible then I agree! I haven't looked into this but if it exists, then we should make a template and that should be the "source of truth"

choldgraf commented on projects/managed-hubs/support.md:
```diff
@@ -1,118 +1,217 @@
+- If the issue is an {term}`Incident`
+  - We will prioritize resolving it over everything else.
+  - We will open a dedicated [Incident issue](support:incident-response) to track its progress.
+  - We will provide regular communication to the {term}`Community Representative` as we investigate and work to resolve the issue.
```
```suggestion
  - We will provide regular communication to the {term}`Community Representative`(s) as we investigate and work to resolve the issue.
```

choldgraf commented:
### Update: added incident commander

After some feedback I've made some more edits with the following main changes:

- Added the **Incident Commander** role
- Separated out separate pages for Support and Incident Response
- Removed our dedicated "Roles for the service" page, and instead defined a "Roles / team structure" section in more specific sections
- General content improvement and clean-up

@yuvipanda want to take a look and we can discuss?

GeorgianaElena reviewed (commented):
I think I like the direction this is going towards and I am curious to see how this will play out in practice, with this process formalized.  I left a few suggestions and questions about things that I didn't understand. Thank for working on this @choldgraf ✨

choldgraf commented on projects/managed-hubs/incidents.md:
```diff
@@ -0,0 +1,107 @@
+   investigate further on next actions. Information about our incident
+   response process can be found [in our team support documentation](https://team-compass.2i2c.org/en/latest/projects/managed-hubs/support.html).
+
+   We've also opened an Incident Report in this issue where you can track progress if you wish: { LINK TO INCIDENT REPORT }.
```
I've updated this per your suggestion below, I hope that's better!

choldgraf commented on projects/managed-hubs/incidents.md:
````diff
@@ -0,0 +1,107 @@
+   ```
+2. **Open an incident issue**.
+   For each {term}`Incident` we create a dedicated issue to track its progress. [{bdg-primary}`open an incident issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=type%3A+Hub+Incident%2Csupport&template=3_incident-report.md&title=%5BIncident%5D+%7B%7B+TITLE+%7D%7D) and notify our engineering team via Slack.
+3. **If you can resolve in 30 minutes**, try doing so.
+4. **If you cannot resolve in 30 minutes**, ping our engineering team and our Project Manager in the {guilabel}`#support-freshdesk` channel so that they are aware of the incident.
````
that makes sense to me - I slightly modified your wording to include the possibility that a person really knows they can't do anything to fix it

choldgraf commented on projects/managed-hubs/incidents.md:
```diff
@@ -0,0 +1,107 @@
+11. **Close the incident ticket**. Once we have confirmation from the community (or no response after 48 working hours), and have filled in the incident {term}`Incident Report`, then close the incident by:
+    - Closing the incident issue on GitHub
+    - Marking the FreshDesk ticket as {guilabel}`Closed`
+
+## Key terms
```
```suggestion

## Handing off Incident Commander status

During an incident, it may be necessary to designate another person to be the Incident Commander.
For example, if it is getting late in the current IC's time zone, they feel burnt out from leading the incident response, or there is someone with better visibility or experience to be the Incident Commander.
This is encouraged and expected, especially for more complex or longer incidents!

To designate another team member as the Incident Commander, follow these steps:

1. **Confirm with them** that they are able and willing to serve as the Incident Commander
2. **Update the Incident Report issue** by updating the Incident Commander name in the top comment.
3. **Notify the team** with a comment in the Incident Report issue.

## Key terms
```

choldgraf commented:
Thanks for those comments @GeorgianaElena ! I believe I've addressed them all and also added in a section about handing off IC status to others. Let me know if you have other thoughts or suggestions!

sgibson91 commented on projects/managed-hubs/support.md:
```diff
@@ -1,118 +1,214 @@
-:::{seealso}
-See [our Team Roles calendar](team-roles-calendar) to know when each team member serves in this role.
-:::
+The primarily goals of the Support Stewards are:
```
```suggestion
The primary goals of the Support Stewards are:
```

GeorgianaElena reviewed (approved):
🚀

yuvipanda commented on projects/managed-hubs/incidents.md:
```diff
@@ -0,0 +1,120 @@
+
+### External communication
+
+- The {term}`Support Steward` team acts as the primary point of communication with external stakeholders like the {term}`Community Representative`s.
```
IMO, specifically for incidents, this should be *optional*. This adds an additional load for the support steward and the incident commander (if they are different people). The IC can request the support steward to act as this if necessary, but otherwise I think we should *not* require this of the support steward. They should be able to delegate an incident to the incident commander, and then by default continue with their existing role. I think if the IC is the source of truth, they should by default be the person who communicates too - otherwise we're adding an entirely new person to this chain, and that is often extremely frustrating during an incident process for everyone involved.

So my suggestion is that the IC can ask someone else to be the point of contact (support steward or someone else) if they so choose to, but that is not the default.

choldgraf replied:
Your reasoning makes sense to me - one concern I have is that the incident commander needs to spend their cycles on resolving the incident, not necessarily also communicating it externally. Maybe the answer is to say the incident commander does this by default, but if they must log-off or are otherwise overwhelmed, they should delegate another team member to provide external communication?

yuvipanda replied:
Exactly right! The default is they do it but if in their judgement delegating it is the right thing - if the extra communication overhead is worth it (as it often is) they can. They just delegate it to someone, not necessarily the support steward.

damianavila replied:
As long as the IC presents itself (or it is actually introduced by the support steward) to the ticket submitter, I think it makes sense NOW to make this optional. I also like the idea to delegate the communication to another one who is not necessarily the support steward (because we do not have a lot of people in support to address other tickets).

Why do I highlight the NOW word? Because when we get a dedicated support team, there should be a clear separation of boxes, IMHO. The support team should be handling the communication with the ticket submitters because they are trained and specialized to interact with people in stress looking for answers. An IC coming from the eng team is not well prepared for that interaction... and that might be a source of issues.

choldgraf replied:
yeah - thinking to the [Incident Commander docs from pagerduty](https://response.pagerduty.com/before/different_roles/) I think this is the "external liason" role. They also have a dedicated person/team to do that work. In our case, I think there are two things I worry about there:

- We don't have the staffing capacity for this now, but maybe this will change in the future as @damianavila suggests.
- If people are not awake at the same time, we pay a big communication penalty when we have bottlenecks of information. If one person must be the one to communicate externally, and that person just went to sleep, then it means no communication can occur until they return to work. This feels like a stressful situation given that we don't have the staffing to ensure seamless handoffs between time-zones all the time.

damianavila replied:
Yep, this is why I think this is fine for now but we should change it in the future when we have enough capacity.

choldgraf replied:
OK I've updated this so that the Incident Commander is the communicator by default, and may delegate if they wish.

yuvipanda commented on projects/managed-hubs/incidents.md:
```diff
@@ -0,0 +1,120 @@
+   For each {term}`Incident` we create a dedicated issue to track its progress. [{bdg-primary}`open an incident issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=type%3A+Hub+Incident%2Csupport&template=3_incident-report.md&title=%5BIncident%5D+%7B%7B+TITLE+%7D%7D) and notify our engineering team via Slack.
+3. **Try resolving the issue** and take notes while you gather information about it.
+4. **If after 30 minutes the issue is not solved or you know you cannot resolve it**, ping our engineering team and our Project Manager in the {guilabel}`#support-freshdesk` channel so that they are aware of the incident.
+5. **Designate an {term}`Incident Commander`**. If the Support Steward wishes to designate someone other than themselves as Incident Commander, do this in the Incident issue.
```
i like this.

damianavila replied:
So, this line implies the support steward has the "power" to designate the IC. How that power will be practiced?
I think there should be some known expectations/details around this. For instance, can the designation be rejected, and what do we do if that happens?

choldgraf replied:
This is a good point - also important to think about the power dynamics here. If the support steward is a brand new team member with less experience than others, they may not feel comfortable just "delegating" to somebody else.

Is this a role that the Project Manager could play? I recall from https://github.com/2i2c-org/infrastructure/issues/1068 that one of the case studies there used a workflow like:

- Support person tries to resolve themselves first
- If they can't, they bring open an issue about this and discuss with the team manager (in our case, I think this would be the project manager)
- Team manager then routes that work item to somebody else on the team.
- Or if it is more complex, they discuss in their next team standup (I believe it is daily for them) and somebody is assigned to that work item out of that meeting

damianavila replied:
> Is this a role that the Project Manager could play?

Yep, probably.
I would suggest still keeping the power to designate in the support steward's hands for the sake of simplicity and quickness... but putting in the PM's hands the tie-breaker "power" is some conflict arises.
I would also encourage the support steward to have a conversation and agreement with the future IC before the designation actually happens.

choldgraf replied:
I'll add a note about asking the Project Manager

yuvipanda commented on projects/managed-hubs/incidents.md:
```diff
@@ -0,0 +1,120 @@
+5. **Designate an {term}`Incident Commander`**. If the Support Steward wishes to designate someone other than themselves as Incident Commander, do this in the Incident issue.
+6. **Investigate and resolve the incident**. The Incident Commander should follow the structure of the incident issue opened in the step above.
+7. **Delegate to Subject Matter Experts as-needed**. The Incident Commander is empowered to delegate actions to Subject Matter Experts in order to investigate and resolve the incident quickly.
+8. **Communicate every few hours**. The {term}`Incident Commander` is expected to communicate incident status and plan with the {term}`Support Steward`s, and the Support Stewards are expected to communicate to the {term}`Community Representative`s. They should provide periodic updates to communities as we attempt to resolve the incident. Here is a template to get started:
```
See comment above about removing support steward from the line of fire here by default. I think this is especially important to us given our diverse timezones that we reduce communciation overhead by default.

yuvipanda reviewed (changes requested):
Thank you very much for writing this up, @choldgraf! I absolutely love it and I think it's an improvement over our current process.

I've left some inline comments about reducing a burden on the support steward when the IC is a person different from the support steward. One of the important outcomes here is to make sure the support stewards don't burn out, and I think the default state when there is a separate IC should be that the support steward is no longer responsible for that (unless they are asked to). For example, if there are multiple ongoing incidents this puts a particularly bigger burden on the support steward. The communication overhead may also be significant, as there's now two extra places where communication needs to happen by default. I recognize that this might vary with individual IC style, but I think the default should be that we don't *require* the support steward to do this. Instead, the IC can call in someone to help with communications - this person can be the support steward, or someone else. I'd rather have us codify that than default to adding another duty to the support steward role.

damianavila commented on projects/managed-hubs/incidents.md:
```diff
@@ -0,0 +1,120 @@
+ 
+   We'll prioritize resolving this incident over our other work, and
+   will communicate with you throughout our attempts to resolve it.
+   We might be in touch with requests for clarifications if needed.
```
Where we are asking for clarifications... I think it should be in the support Freshdesk ticket.
Then, I think we need to enforce the idea the GH issue is for internal communication and for transparency but we should avoid getting information from multiple channels.

choldgraf replied:
Have clarified this

damianavila commented on projects/managed-hubs/incidents.md:
```diff
@@ -0,0 +1,120 @@
+4. **If after 30 minutes the issue is not solved or you know you cannot resolve it**, ping our engineering team and our Project Manager in the {guilabel}`#support-freshdesk` channel so that they are aware of the incident.
+5. **Designate an {term}`Incident Commander`**. If the Support Steward wishes to designate someone other than themselves as Incident Commander, do this in the Incident issue.
+6. **Investigate and resolve the incident**. The Incident Commander should follow the structure of the incident issue opened in the step above.
+7. **Delegate to Subject Matter Experts as-needed**. The Incident Commander is empowered to delegate actions to Subject Matter Experts in order to investigate and resolve the incident quickly.
```
I guess my previous question applies to this line as well 😉.

damianavila commented on projects/managed-hubs/incidents.md:
````diff
@@ -0,0 +1,120 @@
+   to resolve it. We will keep you updated as we continue to make progress.
+   Please let us know if you have had more reports of issues,
+   or reports that your issues have gone away.
+   ```
````
In previous experiences, I have observed periodic updates without "real" updates are usually detrimental to the evolution of the conversation, we should be vigilant about avoiding that pattern.

damianavila replied:
Indeed.

yuvipanda replied:
Your comment is important to us. Please stay on the line.

yuvipanda replied:
Your comment is important to us. Please stay on the line.

yuvipanda replied:
Your comment is important to us. Please stay on the line.

yuvipanda replied:
Your comment is important to us. Please stay on the line.

yuvipanda replied:
Your comment is important to us. Please stay on the line.

damianavila commented on projects/managed-hubs/incidents.md:
```diff
@@ -0,0 +1,120 @@
+2. **Open an incident issue**.
+   For each {term}`Incident` we create a dedicated issue to track its progress. [{bdg-primary}`open an incident issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=type%3A+Hub+Incident%2Csupport&template=3_incident-report.md&title=%5BIncident%5D+%7B%7B+TITLE+%7D%7D) and notify our engineering team via Slack.
+3. **Try resolving the issue** and take notes while you gather information about it.
+4. **If after 30 minutes the issue is not solved or you know you cannot resolve it**, ping our engineering team and our Project Manager in the {guilabel}`#support-freshdesk` channel so that they are aware of the incident.
```
From the PM's perspective, the incident issue should be added to the sprint (cycle) and team backlogs, and the PM should do that.

choldgraf replied:
have added that in!

damianavila commented on projects/managed-hubs/incidents.md:
```diff
@@ -0,0 +1,120 @@
+
+1. **Confirm with them** that they are able and willing to serve as the Incident Commander
+2. **Update the Incident Report issue** by updating the Incident Commander name in the top comment.
+3. **Notify the team** with a comment in the Incident Report issue.
```
OK, I see here a process to look for another IC, I like this!
I also think the first step should also be practiced for the original IC.

choldgraf replied:
I've added a note about this at the top

damianavila commented on projects/managed-hubs/support.md:
```diff
@@ -1,118 +1,214 @@
-We should try to respond to all support-related communications within one working day.
-:::
+- Our [FreshDesk account](https://2i2c.freshdesk.com/a/) allows for internal team communication via the {guilabel}`Add Note` button. This can be useful for sharing quick internal updates.
+- [Issues with the {guilabel}`support` label](https://github.com/2i2c-org/infrastructure/issues?q=is%3Aopen+label%3Asupport+sort%3Aupdated-desc) are where we track support requests related to {term}`Change Request`s and {term}`Guidance Request`s.
```
Maybe adding a note about starting the request via the support email instead of creating an issue on GH.

damianavila commented on projects/managed-hubs/support.md:
```diff
@@ -1,118 +1,214 @@
+
+1. **Monitor our support channels**. We use FreshDesk for all support requests, and the Support Stewards should regularly keep an eye on this account for new requests.
+   When a new support requests comes in, move to the next step.
+2. **Read and understand**. Within 8 working hours, read the support request and try to understand what action would resolve it.
```
I would suggest 24hs here.

damianavila replied:
Ah... OK, this is the reading phase, not the response... then I would suggest 12hs (because of TZs).

choldgraf replied:
Yeah I guess maybe we should use something like "24 non-weekend hours" instead of "8 working hours". I think the general point we want to get across is "within 1 business day" (thus "8 working hours") but this is hard to convey when time-zones mean everybody has a different set of working hours.

Do you think "24 working hours" makes sense then, and maybe we add a note at the top of the document about how we define "working hours" so that others don't think that we actually mean "3 business days" when they read it?

yuvipanda replied:
@choldgraf I definitely read this as 3 working days!

damianavila replied:
> Do you think "24 working hours" makes sense then, and maybe we add a note at the top of the document about how we define "working hours"

Yep, I think we need to clarify the unit (whatever unit we are going to use) at the top so everyone knows what we are talking about 😉.

choldgraf replied:
I tried to add a clarification about this let me know what you think

damianavila commented on projects/managed-hubs/support.md:
````diff
@@ -1,118 +1,214 @@
+   this request into our team's work planning process. We'll send an update
+   when we've got a plan for completing this request.
+   ```
+5. **Prioritize the request**. Work with our team to decide how we should prioritize this request relative to the other work we need to do. Follow the [how to prioritize Change and Guidance Requests guide](support:prioritize-requests).
````
Maybe also talk with the PMs to agree on prioritization levels.

choldgraf replied:
yeah I agree - this is a process that the Project/Product managers should be involved with.

damianavila commented on projects/managed-hubs/support.md:
````diff
@@ -1,118 +1,214 @@
 
-## Who can make support requests?
+```{glossary}
+Support Budget
````
You mentioned above some hours are allocated as Support Budgets, what is the amount we are talking about?

damianavila replied:
I know you added a note above about it... just wondering if you have a number in your head.

choldgraf replied:
I am not sure haha, but here is some thinking about a rationale for arriving at a number.

- Assume 173 working hours a month per engineer
- Assume any given engineer should spend no more than 20% of their time (one day a week) on average dealing with support requests
- That amounts to 34 hours a month on support requests.

Then we can draw this number back to our question "how many hubs should be able to sustain a single engineer?". That ranges between 20 and 8 (depending on edu/research, and dedicated cluster). So as an example, if we use those as limits, we could say that those various hub types correspond to 34/20 ~= 1 hours, or 34 / 8 ~= 4 hours of support each month.

I think that more generally, it'd be better to just more explicitly tie the idea of "more support budget" to "higher monthly cost", but maybe this is a starting point for conversation? Don't think too much about the specific numbers here, I'm wondering if this general approach to arriving at a number makes sense. (also welcome to feedback about specific numbers as well)

damianavila replied:
I think the rationale makes sense as a first iteration. When we have measurements and real data about the time we spend on support, we will have a better understanding and we may tweak/adjust the values (or even the rationale).

choldgraf commented:
OK I believe that I've addressed each of the comments above! Please let me know if that makes sense or if you'd like to see any other edits!

damianavila commented on projects/managed-hubs/incidents.md:
```diff
@@ -0,0 +1,133 @@
+
+To designate another team member as the Incident Commander, follow these steps:
+
+1. **Confirm with them** that they are able and willing to serve as the Incident Commander
```
```suggestion
1. **Confirm with them** that they are able and willing to serve as the Incident Commander.
```

damianavila commented on projects/managed-hubs/support.md:
```diff
@@ -1,118 +1,236 @@
-When a support request is made that requires an action from a 2i2c engineer, a Support Steward should describe this change in a GitHub issue, and add it to the [Sprint Board](coordination:sprint-board).
-Think about an engineering team member that likely has the skills and capacity needed, and ask them if they are willing to take on resolving this issue.
-Try not to ask the same person for support help many times in a row - we should spread the work needed to address support issues across the team.
+  Here is a rough idea of our rationale to follow for arriving at a specific number:
```
You mention arriving at "a specific number" but I think you did not mention that specific number above, is that intended?

From your previous comment, that number would be:

> we could say that those various hub types correspond to 34/20 ~= 1 hours, or 34 / 8 ~= 4 hours of support each month.

I know you can derive that from the rationale you have included but I feel it needs a conclusion like the above one.

choldgraf replied:
I'll try to make it clear that this number is not yet specific, and that we want to keep it as a rationale only, but no precise numbers.

choldgraf commented on projects/managed-hubs/incidents.md:
````diff
@@ -0,0 +1,118 @@
+
+1. **Acknowledge the incident**. Communicate with the Community Representative that there is an incident. Use this canned response as a start for responding:
+
+   ```{button-link} https://2i2c.freshdesk.com/helpdesk/canned_responses/folders/80000143608/responses/80000247490/edit
````
```suggestion
   ```{button-link} https://2i2c.freshdesk.com/a/admin/canned_responses/folders/80000143608/responses/80000247490/edit
```

choldgraf replied:
```suggestion
   ```{button-link} https://2i2c.freshdesk.com/a/admin/canned_responses/folders/80000143608/responses/80000247490/edit
```

choldgraf commented on projects/managed-hubs/incidents.md:
````diff
@@ -0,0 +1,118 @@
+8. **Delegate to Subject Matter Experts as-needed**. The Incident Commander is empowered to delegate actions to Subject Matter Experts in order to investigate and resolve the incident quickly.[^note-on-delegation]
+9. **Communicate our status every few hours**. The {term}`External Liason` is expected to communicate incident status and plan with the {term}`Community Representative`s. They should provide periodic updates that describe the current state of the incident, what we have tried, and our intended next steps. Here is a canned response to get started:
+
+   ```{button-link} https://2i2c.freshdesk.com/helpdesk/canned_responses/folders/80000143608/responses/80000247492/edit
````
```suggestion
   ```{button-link} https://2i2c.freshdesk.com/a/admin/canned_responses/folders/80000143608/responses/80000247492/edit
```

choldgraf commented on projects/managed-hubs/incidents.md:
````diff
@@ -59,26 +59,14 @@ We prioritize the resolution of incidents above all other kinds of work, and hav
-1. **Acknowledge the incident**. Communicate with the Community Representative that there is an incident. Here is a template to get started:
+1. **Acknowledge the incident**. Communicate with the Community Representative that there is an incident. Use this canned response as a start for responding:
 
+   ```{button-link} https://2i2c.freshdesk.com/helpdesk/canned_responses/folders/80000143608/responses/80000247490/edit
````
oops, I think I fixed this

choldgraf commented:
### Question: should this be in docs.2i2c.org/about ?

I made a few more edits to this content, and I find myself naturally wanting to look for it in the "About the service" section of our 2i2c service documentation (somewhere in here: https://docs.2i2c.org/en/latest/about/overview.html). I feel like this section describes the human infrastructure and strategy behind the service, and is a hybrid of documentation for our team (to know what to follow) and other communities (to know what to expect).

Do others agree that this content would make more sense inside docs.2i2c.org ?

If so, I am curious what you think about the *other* content in the "Managed Hubs Service" section [in our Team Compass](https://team-compass.2i2c.org/en/latest/projects/managed-hubs/roles.html). I kind-of feel like this could also live in `docs.2i2c.org` since much of it is 2i2c-facing as well. If that were the case, we could just have pointers in our team compass to those docs.

Curious if others have thoughts

choldgraf commented:
### Updates to this PR

I spoke with @damianavila in particular (and others in passing) about the idea of moving these docs to docs.2i2c.org, and in general the consensus seemed to be that we should keep "2i2c-specific" docs in our team compass to avoid cluttering up the service docs at docs.2i2c.org.

So, the latest commit adds a few more updates and reorganizations to try and lean into this separation of duties a bit more. It does a few main things:

- Removes some sections and definitions here and cross-links to docs.2i2c.org instead, to treat those docs as the source of truth for most things
- Adds some more glossary entries that we can use here and elsewhere
- Tries to reframe our relationship with communities as partnerships rather than customer relationships
- Adds some extra references to our incident response and support models

yuvipanda commented on about/strategy.md:
```diff
@@ -39,15 +39,15 @@ As a start, we wish to launch a **Managed JupyterHubs** service, and will begin
 - 2i2c manages more lightweight, community-specific JupyterHubs for several smaller groups in research and education.
 - 2i2c manages a "generic" JupyterHub that is not tied to any single institution or group.
-- 2i2c has a beta-level business model for the first iteration of our Managed JupyterHub service.
+- 2i2c has a beta-level sustainability model for the first iteration of our Collaborative JupyterHub Service.
```
IMO Managed is a more straightforward and broadly accepted term for what we do, while Collaborative is generally overloaded to the point of being a buzzword. I'd suggest we keep calling it 'Managed'

yuvipanda reviewed (commented):
I love the incident commander stuff, and we can iterate as we move forward.

I'm very confused by 'Collaborative' replacing 'Managed' service though.
